### DISCUSSION TITLE
Allow to scale below original yield

### DISCUSSION THREAD

**Discussion by bartkummel** [mealie-recipes/mealie#3754] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When scaling recipes, I can only scale up. But there are valid use cases for scaling down; e.g. where a recipe yields 4 servings, and you want only 2.

### Please provide a concise description of the feature that would resolve your issue.

When I click the minus button at the moment the recipe is at scale 1, I'd expect the scale to go to 0.5, etc.

### Please consider and list out some caveats or tradeoffs made in your design decision

- You could end up with "impossible" ingredients, like "½ egg".
- There is also #1390. However, I think with the small change I propose here, basically everything is possible, without making things too complicated.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
If you type 0,5 into the scaling field you will get half the recipe yield.

**Reply by bartkummel** [reply]
I know, but using the `–` button is just so much easier and intuitive! BTW: I've a PR (#3755) in the works already.
